examples/run_tests.py

```diff
@@ -88,6 +88,10 @@ def main(examples, break_on_errors=True):
         with working_directory(folder):
 
             params = read_config(file_)
+            # https://github.com/haddocking/haddock3/pull/295#issuecomment-1024057662
+            for p in params:
+                if isinstance(params[p], dict) and p != "topoaa":
+                    params[p]['tolerance'] = 20
```

Comment:
This does not apply tolerance `20%` to the runs. You are changing the `params` dict within the python session but that is **not** passed to the configuration file. You would need to resave the config file again after the change. 

This script only runs `haddock3 <config>` for each of the examples listed.  If you want to change some parameter you need to change it **in the configuration file itself**. There are configuration files for the `tests`; I don't see the need to add a parameter modification in the `run_tests.py`. Also, if you add parameters in the `run_tests.py` overwriting the configuration files then we lose control of the configuration for tests. I disagree with changing the parameter in the `run*.py` files. These files should be just engines to run the configs one after the other, nothing more.

I strongly suggest adding the tolerance **in** the configuration files themselves.